conda/models/match_spec.py

```diff
@@ -166,7 +167,9 @@ class MatchSpec(object):
         'version',
         'build',
         'build_number',
-        'track_features',
```

Comment:
So if I'm reading this right, you're formally removing `track_features` from internal use, translating all of its instances into `provides_features`, but you're not doing the same for `features` by itself. Why not?